src/main/java/mate/controller/carcontroller/DeleteCarController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller.carcontroller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/delete")
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        boolean isCarDeleted = carService.delete(id);
+        if (isCarDeleted) {
+            req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+        }
```

Comment:
don't overcompicate code with redundant checks
```suggestion
        carService.delete(id);
        resp.sendRedirect(req.getContextPath() + "/index");
```